changelog/README.rst

```diff
@@ -0,0 +1,39 @@
+*********************
+Manging the Changelog
+*********************
+
+This directory contains "newsfragments" which are short files that contain a
+small **ReST**-formatted text that will be added to the next ``CHANGELOG``.
+
+The ``CHANGELOG`` will be read by users, so this description should be aimed to
+molecule users instead of describing internal changes which are only relevant
```

Comment:
I think internal changes can still be useful. I saw others putting those under a separate category, like `misc` or `internal`.

OTOH wrt the style, we should encourage fragment authors to follow the same principles as are useful for docstrings in Python or commit messages in Git. For example, use imperative sentences. Describe things on the high-level outlining effects of the changes, not implementation details.

Reply:
Fair points. 

However, for the first attempt, let's just deal with our main priority: a user changelog.